fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/jbatch/export/patient/ChunkReader.java

```diff
@@ -215,19 +217,26 @@ public Object readItem() throws Exception {
             }
 
             resourceType = ModelSupport.getResourceType(ctx.getPartitionResourceType());
-            if (dto != null && !dto.empty()) {
+
+            if (resources != null && !resources.isEmpty()) {
                 if (logger.isLoggable(Level.FINE)) {
                     logger.fine("readItem[" + ctx.getPartitionResourceType() + "]: loaded " + dto.size() + " patients");
                 }
 
-                List<String> patientIds = dto.getResources().stream()
+                List<String> patientIds = resources.stream()
                             .filter(item -> item.getId() != null)
-                            .map(item -> item.getId())
+                            .map(item -> "Patient/" + item.getId())
```

Comment:
I'm planning to remove this in https://github.com/IBM/FHIR/pull/2102

Reply:
OK, please go ahead in 2012, I'd rather leave here since it resolves the issue and ensures a regression doesn't happen.